### PR TITLE
:sparkles: Adds ability to also format the code

### PR DESCRIPTION
**Pull request by rickstaa**
In this pull request, I added the ability also to format the code. This allows us to automatically create a pull request using the [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request) with the required changes. I will add the test report to this pull request.  Let me know what you think and if you have any suggestions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,36 +98,64 @@ jobs:
...
+on: [pull_request]
+jobs:
+  name: runner / black
+  runs-on: ubuntu-latest
+  steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+    - name: Check files using black formatter
+      uses: reviewdog/action-black@v1.1.2
```
Comment (tsuyoshicho) [inline README.md:118]: Use version @v1

File: README.md [hunk 2]
```diff
@@ -77,36 +98,64 @@ jobs:
...
+```yaml
+name: reviewdog
+on: [pull_request]
+jobs:
+  name: runner / black
+  runs-on: ubuntu-latest
+  steps:
+    - uses: actions/checkout@v2
+    - name: Check files using black formatter
+      uses: reviewdog/action-black@v1.1.2
```
Comment (tsuyoshicho) [inline README.md:143]: Same it

File: action.yml [hunk 3]
```diff
@@ -6,7 +6,21 @@ inputs:
     description: "Working directory relative to the root directory."
     required: false
     default: "."
+  format:
+    description: "Don't write the files back, just return the status."
+    required: false
+    default: "false"
+  fail_on_error:
```
Comment (tsuyoshicho) [inline action.yml:13]: What's moved?

If option was related, add more info in description for `format`.

**Review by tsuyoshicho — CHANGES_REQUESTED** [review]
Nice work! and basically good.

See suggest and below point.

- Versioning: up minor with add new param(or major?)
- Fix misspell: "occured" is a misspelling of "occurred"
- README write clearly with 2way reviewdog repot and/or black format

File: action.yml (hunk 3 above)
Comment (tsuyoshicho) [reply action.yml:13]: https://github.com/reviewdog/action-black/blob/da08ab17ca7406548245f2d3940f924ad9ff5056/entrypoint.sh#L75-L78

(note)

File: README.md (hunk 1 above)
Comment (rickstaa) [reply README.md:118]: Good catch!

**Review by tsuyoshicho — APPROVED** [review]
Good!
If you need more improvement, treat code indent/spacing.

**Comment by rickstaa** [comment]
✅ [Tests results (success)](https://github.com/rickstaa/action-test-repo/runs/1617074126?check_suite_focus=true)

**Pull request merged**